### ISSUE TITLE
IA Writer?

### ISSUE THREAD

**Issue by randallb** [errata-ai/vale-server#25]
Any interest in integrating natively with IA Writer?

**Comment by jdkato** [comment]
Yes, this is definitely something I'd be interested in -- does IA Writer offer an API for third-party integrations?

**Comment by ChrisChinchilla** [comment]
@jdkato I wonder if it's worth moving this issue https://github.com/errata-ai/vale/issues/188 to this repo, as it's likely where support will be added? Keeping that commercial / OS tools separation between CLI and server?
